scanpy/plotting/tools/scatterplots.py

```diff
@@ -191,6 +191,10 @@ def plot_scatter(adata,
     else:
         args_3d = {}
 
+    if adata.raw is None and use_raw is True:
```

Comment:
Since a while, the convention in all the plotting functions is that `use_raw=None` is the default (you might remember that I asked you for your opinion a few weeks back). Hence, you can throw an error if someone sets `use_raw=True` and `.raw is None`.

Reply:
I modify this to ajust to the convention.